md/bdm-access-control.md

```diff
@@ -314,3 +315,163 @@ Deploy your access control file.
 Access to your previous data is now controlled by the BDM Access Control file you just deployed. An order picker can't see the customer and the price of a product anymore. A Sales can't see the address of a customer anymore.  
 Connect onto the portal as a user with the profile *Order picker*. Go to your application: the customer data and the order price are always empty.
 Connect now as a user with the profile *Sales*. On your application, the customer data are displayed except the address. The price of the product is now available.
+
+
+### III - Instances protection
+
+Instance protection is not supported as a part of the BDM Access Control feature. However it is still possible to solve most use cases using the [rest-api authorizations](rest-api-authorization.md).
+Following is a step by step guide on how to realize one such (simple) use case:
+
+::: info
+The method shown in the example below is not a new 7.7 feature, it is present in the Community version since 7.0 ( although the location of the files and the push/pull procedure changed in 7.6.0).
+It allows to protect the bdm instances by limiting the access to the BDM query requests that retrieve the object instances rather than to the instances themselves.
```

Comment:
Does it allow to protect or to grant access?
If this is a white list mechanism, then we should make clear this grants access.
I know, the difference is tiny, but this sets users' mind about the white list way to use the feature.

Reply:
"It allows to protect the bdm instances" -> that's what the user tries to do.

As for the way it works, it's literally one/several scripts that return true or false for a query. Depending on what you put in those scripts, you get either a black or white list.